packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -190,21 +209,15 @@ const DatePicker = React.memo(props => {
         <React.Fragment>
             <MobileWrapper>
                 <Native
+                    {...common_props}
                     id={id}
                     name={name}
-                    display_format={display_format}
-                    error={error}
-                    label={label}
-                    max_date={max_date}
-                    min_date={min_date}
                     onBlur={onBlur}
                     onFocus={onFocus}
                     onSelect={onSelectCalendarNative}
-                    placeholder={placeholder}
                     value={getCalendarValue(date)} // native picker accepts date format yyyy-mm-dd
                     disabled={disabled}
                     data_testid={data_testid}
-                    {...common_props}
```

Comment:
@amir-deriv We should keep the prop spreading at the end, So we can override the existing props with the one coming from outside, This change might cause unexpected behavior 🤔